kitsune/sumo/jinja2/includes/common_macros.html

```diff
@@ -184,12 +214,66 @@ <h4 class="mzp-c-menu-item-title">{{ _('Explore by topic') }}</h4>
               <h4 class="mzp-c-menu-item-title">{{ _('Browse by product') }}</h4>
             </div>
             <ul class="mzp-c-menu-item-list sumo-nav--sublist">
-              <li><a href="{{ url('questions.list', 'firefox')}}">{{ _('Firefox browser') }}</a></li>
-              <li><a href="{{ url('questions.list', 'focus-firefox') }}">{{ _('Firefox Focus') }}</a></li>
-              <li><a href="{{ url('questions.list', 'ios') }}">{{ _('Firefox for iOS') }}</a></li>
-              <li><a href="{{ url('questions.list', 'thunderbird') }}">{{ _('Thunderbird') }}</a></li>
-              <li><a href="{{ url('questions.list', 'mobile') }}">{{ _('Firefox for Android') }}</a></li>
-              <li><a class="color-link" href="{{ url('questions.home') }}">{{ _('View all forums') }}</a></li>
+              <li>
+                <a href="{{ url('questions.list', 'firefox')}}"
+                   data-event-name="link_click"
+                   data-event-parameters='{
+                    "link_name": "main-menu.question-list",
```

Comment:
If we want to be consistent with what I've done before for the `link_name` parameter, we should use either `main-menu.{slugified text of menu item}.{content_group of target}` or `main-menu.{content_group of target}` -- so maybe we should adjust this to:

```suggestion
                    "link_name": "main-menu.support-forum-home",
```